site/content/en/faq.md

```diff
@@ -388,46 +253,334 @@ information, or simply redirect the reader to your `LICENSES/` directory.
+
+Before you proceed, always first make sure that you can find the copyright and
+licensing information elsewhere. Some projects only include this information in
+the root directory or in their README file.
+
+If you can find no copyright notice, you can try to add it yourself by divining
+the copyright holder from the author. See [this
+question](#copyright-holder-author) for more information on the distinction.
+When in doubt, contact the author for clarification.
+
+If the work has no license, then that means that you do not have the right to
+copy it. If you believe that this is a mistake and the author clearly meant for
+you to be able to copy this work, you should contact the author and ask them to
+license their work. Feel free to refer them to <https://reuse.software>.
+
+## ... exclude a file from REUSE compliance testing? {#exclude-file}
+
+If the file is a build artifact and you use Git, simply make sure that the file
+is covered by your `.gitignore` file. The same goes for other VCS ignore
+mechanisms.
+
+Otherwise, you cannot exclude files from REUSE compliance testing. It would go
+entirely against the purpose of REUSE: making sure that every file has a
+copyright and license tag.
+
+If you really want to exclude a file, consider using the
+[CC0-1.0](https://creativecommons.org/publicdomain/zero/1.0/) license for this
+file. By doing this, you put the file in the [public domain, or your country's
+equivalent](https://fsfe.org/freesoftware/legal/faq.html#public-domain).
+
+If you have an entire directory that you want to "exclude" from REUSE compliance
+testing, you can [use `REUSE.toml`](#bulk-license).
+
+## ... exclude certain lines from REUSE compliance testing? {#exclude-lines}
+
+In order to make the tool ignore a specific section containing strings that may
+falsely be detected as copyright or license statements, you can wrap it within
+the two comments `REUSE-IgnoreStart` and `REUSE-IgnoreEnd`.
+
+<!-- REUSE-IgnoreStart -->
+
+Please note that this must not be used to ignore valid copyright and licensing
+information by yourself or a third party. The ignore blocks must only be used
+for marking blocks that may trigger false-positive detections and errors.
+
+An example for a file that contains commands or documentation that confuse the
+REUSE tool:
+
+```bash
+# SPDX-FileCopyrightText: 2021 Jane Doe
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+echo "SPDX-FileCopyrightText: $(date +'%Y') John Doe" > file.txt
+echo "SPDX-License-Identifier: MIT" >> file.txt
 ```
 
-You can read more about this file format by Debian
-[here](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/).
+This is how the section can be ignored:
+
+```bash
+# SPDX-FileCopyrightText: 2021 Jane Doe
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# REUSE-IgnoreStart
+echo "SPDX-FileCopyrightText: $(date +'%Y') John Doe" > file.txt
+echo "SPDX-License-Identifier: MIT" >> file.txt
+# REUSE-IgnoreEnd
+```
+
+<!-- REUSE-IgnoreStart -->
```

Comment:
```suggestion
<!-- REUSE-IgnoreEnd -->
```

…or am I missing something?

Reply:
this entire document is wrapped in REUSE ignore tags :) but the question on ignore tags keeps closing the document-wide ignore instruction, so we have to re-open it several times.